include/aa_py_filterbank_metadata.hpp

```diff
@@ -21,6 +21,7 @@ namespace astroaccelerate {
 	int m_nbits;
 	int m_nsamples;
 	int m_nchans;
+	double m_src_raj;
```

Comment:
I'm not sure what that should be. It would be best to rename it.

Reply:
I believe is the filterbank part. It should be
J2000 Right Ascension
J2000 Declination